pkg/apisix/cluster.go

```diff
@@ -585,19 +603,22 @@ func (c *cluster) updateResource(ctx context.Context, url string, body io.Reader
 	return &ur, nil
 }
 
-func (c *cluster) deleteResource(ctx context.Context, url string) error {
+func (c *cluster) deleteResource(ctx context.Context, url, resource string) error {
 	req, err := http.NewRequestWithContext(ctx, http.MethodDelete, url, nil)
 	if err != nil {
 		return err
 	}
+	start := time.Now()
 	resp, err := c.do(req)
+	c.metricsCollector.RecordAPISIXLatency(time.Now().Sub(start))
```

Comment:
We may add another label to distinguish the type of operation, like `create`, `update`.